src/views/v2/summary/event/date.php

```diff
@@ -23,21 +23,22 @@
 $event->schedule_details;
 $formatted_start_date = $event->dates->start->format( Dates::DBDATEFORMAT );
 $formatted_group_date = $group_date->format( Dates::DBDATEFORMAT );
-
 ?>
-<div class="tribe-events-calendar-summary__event-datetime-wrapper tribe-common-b2">
+<div class="tribe-events-calendar-summary__event-datetime-wrapper tribe-common-b3">
 	<?php $this->template( 'summary/event/date/featured' ); ?>
+	<?php $this->template( 'summary/event/date/virtual' ); ?>
 	<time class="tribe-events-calendar-summary__event-datetime" datetime="<?php echo esc_attr( $formatted_start_date ); ?>" title="<?php echo $event->start_date . ' :: ' . $event->end_date; ?>">
 		<?php if ( ! $event->multiday ) : ?>
 			<span class="tribe-event-date-start"><?php echo esc_html( $event->summary_view->start_time ); ?></span> -
 			<span class="tribe-event-date-end"><?php echo esc_html( $event->summary_view->end_time ); ?></span>
-		<?php elseif ( $event->summary_view->start_date === $formatted_group_date ) : ?>
+		<?php elseif ( ! empty( $event->summary_view ) && $event->summary_view->start_date === $formatted_group_date ) : ?>
```

Comment:
As discussed - this sometimes throws a warning, so we now check `empty()` before checking the value.